README.md

```diff
@@ -0,0 +1,102 @@
+# euphony-listener
+​
+## Introduction
+​
+'Euphony-Listener' receives and recognizes data through sound waves from 'Speaker'. </br>
+The frequency occurs when the speaker sends data. The Listen button is pressed, 'Listener' recognizes the data and you can check the results. </br>
+It only works on android. </br>
+​
+## Euphony Library
+​
+Acoustic Data Telecommunication Library. This is for Android version. </br>
+Euphony provides a handiness library designed to communicate with other devices(android and web) using mic and recorder. </br>
+​
+Official Facebook Page : https://www.facebook.com/euphonyproject </br>
+Official Library Site : https://dev.jbear.co/euphony </br>
+​
+## Prerequisite
+build.gradle in app module </br>
+```java
```

Comment:
```gradle
dependencies {
    implementation 'euphony.lib:euphony:0.7.1.6'
}
```

As you know, here is gradle script.
Please modify language name for code-highlight :)

Reply:
What I mean was like below
```suggestion
```gradle
```

When it comes to line no 19, the previous commit seems better.

Reply:
Sorry for my mistake, changed code-highlight in both README

Reply:
And also reset lastest commit